### ISSUE TITLE
Player Data could not be saved The_Vault 1.6 Server

### ISSUE THREAD

**Issue by echtelerp** [Iskallia/Vault-public#486]
**Describe the bug**
when changing the Mod file on a Server install to 1.6, the server says could not save player Data and kicks the player. 
With 1.5 installed it works as expected. 

**To Reproduce**
1. running server with 1.5 
2. stop server, 
3. copy 1.6 The Vault mod Jar file from client to Server, 
4. Start server again, and try to log in. 

**Expected behavior**
as with previos installs, it should just be running, 

**Screenshots or Crash Report**

**Info (please complete the following information):**
- World Type: [Server]
- The_Vault Version [e.g. 1.6p1.8.10.1.jar]

**Additional context**
What I also noticed: the .jar file size changed from 34.426 KB on 1.5x to 18.300KB on 1.6? 
did you remove half the Code? or is that the problem?

**Comment by echtelerp** [comment]
is closed, had to update also the conf folder of VH, but than evertime we try to open a Vault the server Crashes we simply went back to 1.5 and will wait for a new server pack for 1.6x

**Comment by jmilthedude** [comment]
For this crash when opening a vault, a crash report is required to be able to help. About the significant decrease in file size, a lot of legacy code was in fact removed.

**Comment by echtelerp** [comment]
Funny thing, the Server does not create a Crash report. 
it just freezes, and i have to close the window using the X. 
/stop does not work. 
And when I try to reconnect (client) than it does not let me in. And it says on the overview page in MC, that there are still players online, even if they are not :-)

**Comment by BONNe** [comment]
That behaviour was sounds similar to jmt_mcmt mod issue.

Could you check that that mod is not in your mods folder?

**Comment by echtelerp** [comment]
Do you  have a issue # for me?

**Comment by BONNe** [comment]
#398

**Comment by Anaerin** [comment]
I'm getting the same issue. There is no crash report (Crash report handler appears to be crashing with "Negative index in crash report handler"), so I have attached the relevant portion of the debug log, with IP addresses, UUIDs and Player names redacted.
[Debug Portion.log](https://github.com/Iskallia/Vault-public/files/7385292/Debug.Portion.log)
I hope this helps, in the meantime I rolled the jar back to the version in the 1.5.1 modpack (Which is where it started) and it's all working again.
